keylime/cloud_verifier_tornado.py

```diff
@@ -105,6 +105,23 @@
 }
 
 
+# TODO(kaifeng) Move this to db as well as other db objects
```

Comment:
Could you make an issue please, rather then mark todo in the code (that way we have it trackable in the project board).

Reply:
ack, I will remove it on next push.